packages/@aws-cdk/aws-appmesh/test/test.route.ts

```diff
@@ -1,9 +1,200 @@
+        meshName: 'test-mesh',
+      });
+      const router = new appmesh.VirtualRouter(stack, 'router', {
+        mesh,
+      });
+
+      // WHEN
+      const node = mesh.addVirtualNode('test-node', {
+        dnsHostName: 'test',
+        listener: {
+          portMapping:
+            {
+              port: 8080,
+              protocol: appmesh.Protocol.HTTP,
+            },
+        },
+      });
+
+      router.addRoute('test-http-route', {
+        routeSpec: appmesh.RouteSpec.http({
+          weightedTargets: [
+            {
+              virtualNode: node,
+            },
+          ],
+        }),
+      });
+
+      router.addRoute('test-http2-route', {
+        routeSpec: appmesh.RouteSpec.http2({
+          weightedTargets: [
+            {
+              virtualNode: node,
+            },
+          ],
+        }),
+      });
+
+      router.addRoute('test-tcp-route', {
+        routeSpec: appmesh.RouteSpec.tcp({
+          weightedTargets: [
+            {
+              virtualNode: node,
+            },
+          ],
+        }),
+      });
+
+      // THEN
+      expect(stack).to(haveResourceLike('AWS::AppMesh::Route', {
+        Spec: {
+          HttpRoute: {
+            Action: {
+              WeightedTargets: [
+                {
+                  VirtualNode: {
+                    'Fn::GetAtt': [
+                      'meshtestnodeF93946D4',
+                      'VirtualNodeName',
+                    ],
+                  },
+                  Weight: 1,
+                },
+              ],
+            },
+            Match: {
+              Prefix: '/',
+            },
+          },
+        },
+        RouteName: 'test-http-route',
+      }));
+
+      expect(stack).to(haveResourceLike('AWS::AppMesh::Route', {
+        Spec: {
+          Http2Route: {
+            Action: {
+              WeightedTargets: [
+                {
+                  VirtualNode: {
+                    'Fn::GetAtt': [
+                      'meshtestnodeF93946D4',
+                      'VirtualNodeName',
+                    ],
+                  },
+                  Weight: 1,
+                },
+              ],
+            },
+            Match: {
+              Prefix: '/',
+            },
+          },
+        },
+        RouteName: 'test-http2-route',
+      }));
+
+      expect(stack).to(haveResourceLike('AWS::AppMesh::Route', {
+        Spec: {
+          TcpRoute: {
+            Action: {
+              WeightedTargets: [
+                {
+                  VirtualNode: {
+                    'Fn::GetAtt': [
+                      'meshtestnodeF93946D4',
+                      'VirtualNodeName',
+                    ],
+                  },
+                  Weight: 1,
+                },
+              ],
+            },
+          },
+        },
+        RouteName: 'test-tcp-route',
+      }));
+
+      test.done();
+    },
+    'should have expected features'(test: Test) {
```

Comment:
There should be an empty line between tests on the same level:

```suggestion
    },

    'should have expected features'(test: Test) {
```

Same comment everywhere in the tests.